addons/sale_loyalty/models/sale_order.py

```diff
@@ -288,9 +288,8 @@ def _discountable_order(self, reward):
         discountable_per_tax = defaultdict(float)
 
         AccountTax = self.env['account.tax']
-        order_lines = self.order_line.filtered(lambda x: not x.display_type)
         base_lines = []
-        for line in order_lines:
+        for line in lines:
```

Comment:
```suggestion
        for line in lines.filtered(lambda x: not x.display_type):
```
We still don't want to compute taxes for these IMHO =]

Reply:
Actually, never mind, this wouldn't work for for payment programs :upside_down_face:

Reply:
Haha, exactly what I was going to say. Though the lines could be filtered out before these lines too.